src/ui/camera.js

```diff
@@ -609,6 +617,113 @@ class Camera extends Evented {
         return options;
     }
 
+    _minimumAABBFrustumDistance(tr: Transform, aabb: Aabb): number {
+        const aabbW = aabb.max[0] - aabb.min[0];
+        const aabbH = aabb.max[1] - aabb.min[1];
+        const aabbAspectRatio = aabbW / aabbH;
+        const selectXAxis = aabbAspectRatio > tr.aspect;
+
+        const min = aabb.min;
+        const max = [
+            selectXAxis ? aabb.max[0] : aabb.min[0],
+            selectXAxis ? aabb.min[1] : aabb.max[1],
+            aabb.min[2]
+        ];
+        const axisLength = vec3.length(vec3.sub([], min, max));
```

Comment:
Looks like this code is mostly redundant, since `max[2] - min[2]` is always `0`, and two other components are mutually exclusive, so the vector is axis-aligned.

```suggestion
        const axisLength = selectXAxis ? aabbW : aabbH;
```

Reply:
Yep, good catch